static/js/code-highlighter.js

```diff
@@ -496,3 +496,20 @@ $(function () {
   });
 
 });
+
+$(function() {
+  // We use user-select:none to hide line numbers from text selections. That
+  // does however not work in Chrome - https://crbug.com/850685
+  // As a work-around, we will use generated content to render the line numbers.
+  if (navigator.userAgent.indexOf("Firefox") != -1) {
+    // Work-around is only needed in Chrome.
+    return;
+  }
+  $('#file').one('selectstart', function() {
```

Comment:
This seems like a typo that accidentally worked.  I understand `one` to be akin to passing `once: true` to addEventListener, but I presume this would want to happen every time.

Note: I'm not necessarily endorsing this approach or even fixing this bug, just doing a drive-by.

Reply:
Not a typo. https://api.jquery.com/one/

Reply:
Ah, I see, you're transforming the entire DOM the first time someone starts to select something.  It'd be good to have a comment to that effect and possibly switch to using addEventListener directly with `{ once: true }`.  (We're planning to rip out jQuery.)